src/res/KM_Resource.pas

```diff
@@ -216,6 +218,37 @@ procedure TKMResource.LoadGameResources(aAlphaShadows: Boolean);
 end;
 
 
+function TKMResource.GetHouseNotOccupiedMsgId(aHouseType: THouseType): Integer;
+begin
+  case aHouseType of
```

Comment:
This "case" is a bit superfluous, maybe we can do the usual house order trick?
`TX_MSG_HOUSE_UNOCCUPIED_ + Ord(HouseType)`